source/renderer/zettlr-renderer.js

```diff
@@ -79,6 +81,7 @@ class ZettlrRenderer {
         this.beginSearch(term)
       }
     }
+    setTimeout(() => icons.loadCustomIcons(), 0)
```

Comment:
This can be moved to the init-function where the initial configuration is pulled from main, because I thiiiink that this is only called once … but better check twice …